parquet-column/src/test/java/org/apache/parquet/schema/TestTypeBuilders.java

```diff
@@ -1423,6 +1423,32 @@ public void testDecimalLogicalTypeWithDeprecatedPrecision() {
     Assert.assertEquals(expected, actual);
   }
 
+  @Test
+  public void testTimestampLogicalTypeWithUTCParameter() {
+    PrimitiveType utcMillisExpected = new PrimitiveType(REQUIRED, INT64, "aTimestamp",
+      LogicalTypeAnnotation.timestampType(true, LogicalTypeAnnotation.TimeUnit.MILLIS));
```

Comment:
I think, it would be more readable by using static imports. e.g. `timestampType(true, MILLIS)`